gph/src/ripser.h

```diff
@@ -1564,11 +1587,9 @@ class ripser
                       std::greater<>());
 #endif
             for (size_t i = 0; i < idx_essential; ++i) {
-                if (!std::isinf(essential_pair[i])) {
```

Comment:
This is about checking ensuring that births are not infinite ("the bar is never born"), but in this proposal it is not needed because `is_born` makes sure we don't end up in this case.